spec/system/support_interface/add_audit_comment_spec.rb

```diff
@@ -4,7 +4,7 @@
   include DfESignInHelpers
 
   around do |example|
-    Timecop.freeze(Time.zone.local(2019, 10, 1, 12, 0, 0)) do
+    Timecop.travel(Time.zone.local(2019, 10, 1, 12, 0, 0)) do
       example.run
```

Comment:
Do we need this if we are no longer testing for the date an audit was created?

Reply:
We don't, removing...